packages/account/src/Components/poinc/status/not-required/__tests__/not-required.spec.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import { PoincNotRequired } from '../not-required';
+
+describe('<PoincNotRequired/>', () => {
+    it('should render PoincNotRequired component', () => {
+        render(<PoincNotRequired />);
+        expect(screen.getByText('Proof of income verification is not required')).toBeInTheDocument();
```

Comment:
Suggestion: we can write tests to check if element is on the screen based on element ID, rather than text. Writing test cases based on text multiplies the amount of work if we need to change this text in the future because the text has to be changed in multiple test cases as well.